docs/sources/installation/debian.md

```diff
@@ -58,8 +58,10 @@ which is officially supported by Docker.
  
         $ sudo apt-get update
         $ sudo apt-get install -t wheezy-backports linux-image-amd64
+        
+2. Restart your system so Debian will use the new kernel from wheezy-backports
```

Comment:
Missing a period on this.  While you are in there, the pattern is generally action followed by result or system response like this: 

2. Restart your system.
    This is necessary for Debian to use your new kernel.